test/test.py

```diff
@@ -75,7 +75,11 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files, age_range, has
         self.assertEqual(str(bundle.uuid), uuid)
         self.assertEqual(bundle.version, version)
         self.assertEqual(1, len(bundle.projects))
-        self.assertEqual({Project}, {type(e) for e in bundle.projects.values()})
+        project = list(bundle.projects.values())[0]
+        self.assertEqual(Project, type(project))
+        self.assertEqual({contributor.laboratory for contributor in project.contributors if contributor.laboratory},
```

Comment:
You were right. This makes no sense. It just duplicates production code.

```
self.assertEqual(len(project.contributors), len(project.laboratory_names))
```

?